### PR TITLE
Release 0.0.34 hotfix environment release

### PR DESCRIPTION
**Pull request by Anubha98**
Changes:
1. Deploying the same environment as 39 version as 41: https://ml.azure.com/registries/azureml/environments/acft-hf-nlp-gpu/version/39?tid=72f988bf-86f1-41af-91ab-2d7cd011db47. This is due to package conflicts occurring sue to transformers version upgrade. Once the release 0.0.34 (hotfix) will complete, will publish new env again to get current latest 40 env as 42.
2. mltable version upgrade from 1.5.0 to 1.6.1
3. incorporate finetune.py script changes as per AzureMlCli master

Succeeded dev environment: https://ml.azure.com/environments/acft-hf-nlp-gpu/version/before.pypi_0034.5?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

### CODE REVIEW

File: assets/training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #PTCA image
-FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch201:{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch201:biweekly.202401.2
```
[the remark this answers is not in the document]
Comment (skanakamedal) [reply assets/training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile:2]: No need to update this

**Comment by skanakamedal** [comment]
I guess you missed a change to copy conda file in model_selector.py. Did you already pushed it?

**Review by skanakamedal — CHANGES_REQUESTED** [review]

File: assets/training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (Anubha98) [reply assets/training/finetune_acft_hf_nlp/environments/acpt/context/Dockerfile:2]: missed this hardcoding to revert, fixed.

**Review by jpmann — APPROVED** [review]

**Comment by Anubha98** [comment]
> :{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}

**Comment by Anubha98** [comment]
> I guess you missed a change to copy conda file in model_selector.py. Did you already pushed it?

Thats under model_selector_utils.py
image

**Review by skanakamedal — APPROVED** [review]

**Pull request merged**